napari/layers/points/points.py

```diff
@@ -1900,6 +1899,9 @@ def add(self, coords):
             Point or points to add to the layer data.
         """
         self.data = np.append(self.data, np.atleast_2d(coords), axis=0)
+        self.events.data(
+            value=self.data, action_type={"action": "add", "index": [-1]}
```

Comment:
I definitely prefer multiple kwargs rather than the dictionary at `event.value`, but I remember at some stage in discussion with @jni him not being a fan of it (maybe I'm wrong though). 

I think if we're providing multiple args it should be:

```python
self.events.data(
    value=self.data,
    action="add",
    index=[-1]
)
```

Reply:
totally don't remember that 😂 but it's plausible. 😂 either way I think @Czaki's concern of backwards compatibility is a good one, especially when it's easy to satisfy it.

Reply:
It's easy, but also the solution of having a `list` as index seems to cause issues because `QModelIndex` can't accept a list as argument.